Components/navigation.css

```diff
@@ -0,0 +1,239 @@
+.nav__bar {
```

Comment:
the naming convention can be kept consistent. Above files, you didn't use BEM.

Reply:
yeah I will correct that in future